src/test/java/TestUtils.java

```diff
@@ -0,0 +1,39 @@
+
+import java.awt.Component;
+import java.awt.Container;
+
+ /**
+ * TestUtils.java impolements helper methods for testing.
+ * @author Erik Cole
+ * @author Chris Waldon
+ * @version 0.0.1
+ *
+ * Inspired by:
+ * http://www.javaworld.com/article/2073056/swing-gui-programming/
+ * automate-gui-tests-for-swing-applications.html
+ */
+public class TestUtils {
```

Comment:
We do not currently use it.  @ejcole if we follow Fenwick's advice (see previous comment) then we won't use it.  What do you think?